assets/aml-benchmark/tests/test_evaluation_utils.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests for common evaluation utilities."""
+
+import pytest
+
+from aml_benchmark.utils.evaluation_utils import extract_python_function, extract_text_from_markdown_tag
```

Comment:
This import should fail if the package isn't installed, which mostly is the case. We can fix this for everyone by adding the following statements in `./assets/aml-benchmark/tests/__init__.py`:
```
import sys

from .test_utils import get_src_dir


path = get_src_dir()
if path not in sys.path:
    sys.path.append(path)

```